### PR TITLE
add run type test ci

### PR DESCRIPTION
**Pull request by ebattat**
Adding test ci dedicate folder for the test CI, also update dedicate elasticsearch indexes
Update manifest with the dedicate folder and file
The purpose is to have separate elastic indexes for functional ci and test ci.

### CODE REVIEW

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]
I suggest using a run type other than `ci`, as there are already `perf_ci` and `func_ci`.  Prefix the `ci` with something describing the use of this.

File: benchmark_runner/benchmark_operator/workload_flavors/ci/hammerdb/hammerdb_data_template.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  es_index_name: hammerdb-ci
+  # enable/disable system metrics
+  system_metrics: {{ system_metrics }}
+  prom_url: https://prometheus-k8s.openshift-monitoring.svc.cluster.local:9091
+  prom_token: {{ prom_token }}
+  sm_index_name: system-metrics
+  metrics_profile: node-metrics.yml
+  #metrics_profile: https://raw.githubusercontent.com/cloud-bulldozer/benchmark-operator/master/roles/kube-burner/files/metrics.yaml
+mariadb:
+  db_image: centos8-mariadb103-container-disk:latest
```
[the remark this answers is not in the document]
Comment (ebattat) [reply benchmark_runner/benchmark_operator/workload_flavors/ci/hammerdb/hammerdb_data_template.yaml:19]: I gave it the corresponding github workload name:
https://github.com/redhat-performance/benchmark-runner/tree/main/.github/workflows

[the remark this answers is not in the document]
Comment (RobertKrawitz) [reply benchmark_runner/benchmark_operator/workload_flavors/ci/hammerdb/hammerdb_data_template.yaml]: What function is this used to perform?

[the remark this answers is not in the document]
Comment (ebattat) [reply benchmark_runner/benchmark_operator/workload_flavors/ci/hammerdb/hammerdb_data_template.yaml]: I used it to run the test_ci as environment variable:
https://github.com/redhat-performance/benchmark-runner/pull/233/commits/37a12a215eeb80d8c007897733b0564fc6ba919e#diff-7ca690ca03bde98dc29253ba4845da57a35fc2413d5a2a8f039110147a1b49a3R108

[the remark this answers is not in the document]
Comment (RobertKrawitz) [reply benchmark_runner/benchmark_operator/workload_flavors/ci/hammerdb/hammerdb_data_template.yaml]: What's the purpose (human description) of this function?  Functional CI is the CI for benchmark-runner; perf CI is the CI job this runs; what is test CI for?

[the remark this answers is not in the document]
Comment (ebattat) [reply benchmark_runner/benchmark_operator/workload_flavors/ci/hammerdb/hammerdb_data_template.yaml]: @RobertKrawitz let me explain you:
Test_CI - Run test CI against each PR merge
Func_CI -  Run nightly CNV CI against functional env
Perf_CI -  Run nightly CNV CI against performance env

[the remark this answers is not in the document]
Comment (RobertKrawitz) [reply benchmark_runner/benchmark_operator/workload_flavors/ci/hammerdb/hammerdb_data_template.yaml]: OK, then test_CI is a good name for it.  Just need to make sure that the names reflect the functions.

File: benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/mariadb_ephemeral_template.yaml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      labels:
+        app: mariadb
+        type: mariadb-database-server
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: "{{ pin_node2 }}"
+      terminationGracePeriodSeconds: 10
+      containers:
+      - name: mariadb
+        image: centos/mariadb-103-centos8:latest
```
Comment (RobertKrawitz) [inline benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/mariadb_ephemeral_template.yaml:48]: Please fix (repo name)

File: benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/mariadb_ocs_pvc_template.yaml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      labels:
+        app: mariadb
+        type: mariadb-database-server
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: "{{ pin_node2 }}"
+      terminationGracePeriodSeconds: 10
+      containers:
+      - name: mariadb
+        image: centos/mariadb-103-centos8:latest
```
Comment (RobertKrawitz) [inline benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/mariadb_ocs_pvc_template.yaml:48]: Please fix (repo name)

File: benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/postgres_ephemeral_template.yaml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+      labels:
+        app: postgres
+        type: postgres-database-server
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: "{{ pin_node2 }}"
+      terminationGracePeriodSeconds: 10
+      containers:
+        - name: postgres
+          image: centos/postgresql-10-centos8:latest
```
Comment (RobertKrawitz) [inline benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/postgres_ephemeral_template.yaml:40]: Please fix repo name.

File: benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/postgres_ocs_pvc_template.yaml [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+      labels:
+        app: postgres
+        type: postgres-database-server
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: "{{ pin_node2 }}"
+      terminationGracePeriodSeconds: 10
+      containers:
+        - name: postgres
+          image: centos/postgresql-10-centos8:latest
```
Comment (RobertKrawitz) [inline benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/postgres_ocs_pvc_template.yaml:40]: Please fix repo name

File: benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/mssql_ephemeral_template.yaml [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    matchLabels:
+      app: mssql
+  template:
+    metadata:
+      labels:
+        app: mssql
+        type: mssql-database-server
+    spec:
+      nodeSelector:
+              kubernetes.io/hostname: "{{ pin_node2 }}"
```
Comment (RobertKrawitz) [inline benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/mssql_ephemeral_template.yaml:24]: Excess indentation

**Review by RobertKrawitz — COMMENTED** [review]
A few more

File: benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/postgres_ocs_pvc_template.yaml (hunk 5 above)
Comment (ebattat) [reply benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/postgres_ocs_pvc_template.yaml]: Done

File: benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/postgres_ephemeral_template.yaml (hunk 4 above)
Comment (ebattat) [reply benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/postgres_ephemeral_template.yaml]: Done

File: benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/mariadb_ocs_pvc_template.yaml (hunk 3 above)
Comment (ebattat) [reply benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/mariadb_ocs_pvc_template.yaml]: Done

File: benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/mariadb_ephemeral_template.yaml (hunk 2 above)
Comment (ebattat) [reply benchmark_runner/benchmark_operator/workload_flavors/test_ci/hammerdb/internal_data/mariadb_ephemeral_template.yaml]: Done

**Comment by RobertKrawitz** [comment]
/lgtm
/approve

Please squash and merge.

**Review by RobertKrawitz — APPROVED** [review]

**Pull request merged**